{
    "source": "BDSA",
    "name": "BDSA-2021-1709",
    "title": "PoDoFo Vulnerable to Denial-of-Service (DoS) via User-After-Free in 'PoDoFo::PdfVecObjects::Clear' Functionality",
    "description": "PoDoFo contains a use-after-free vulnerability that is generated by flawed program logic. Attackers could carry out a denial-of-service (DoS) attack by providing the application with a maliciously crafted file.",
    "technicalDescription": "This vulnerability exists in the `PoDoFo::PdfVecObjects::Clear` functionality. Attackers could exploit this by supplying the application with a maliciously crafted PDF file triggering an out-of-bounds read, causing a DoS condition.",
    "publishedDate": "2021-06-09T10:13:43.847Z",
    "updatedDate": "2022-06-15T16:11:24.388Z",
    "disclosureDate": "2021-04-03T00:00:00.000Z",
    "exploitPublishDate": "2021-04-03T00:00:00.000Z",
    "solution": "",
    "severity": "MEDIUM",
    "cvss2": {
        "baseScore": 5.0,
        "impactSubscore": 2.9,
        "exploitabilitySubscore": 10.0,
        "severity": "MEDIUM",
        "accessVector": "NETWORK",
        "accessComplexity": "LOW",
        "authentication": "NONE",
        "confidentialityImpact": "NONE",
        "integrityImpact": "NONE",
        "availabilityImpact": "PARTIAL",
        "temporalMetrics": {
            "exploitability": "PROOF_OF_CONCEPT",
            "remediationLevel": "UNAVAILABLE",
            "reportConfidence": "CONFIRMED",
            "score": 4.5
        },
        "vector": "(AV:N/AC:L/Au:N/C:N/I:N/A:P/E:POC/RL:U/RC:C)"
    },
    "cvss3": {
        "baseScore": 5.3,
        "impactSubscore": 1.4,
        "exploitabilitySubscore": 3.9,
        "severity": "MEDIUM",
        "attackVector": "NETWORK",
        "attackComplexity": "LOW",
        "confidentialityImpact": "NONE",
        "integrityImpact": "NONE",
        "availabilityImpact": "LOW",
        "privilegesRequired": "NONE",
        "scope": "UNCHANGED",
        "userInteraction": "NONE",
        "temporalMetrics": {
            "exploitability": "PROOF_OF_CONCEPT",
            "remediationLevel": "UNAVAILABLE",
            "reportConfidence": "CONFIRMED",
            "score": 5.0
        },
        "vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:L/E:P/RL:U/RC:C"
    },
    "useCvss3": true,
    "classifications": [],
    "zeroDay": false,
    "underReview": true,
    "parentAdvisory": false,
    "workaround": "",
    "bdsaTags": [],
    "overallScore": 5.0,
    "_meta": {
        "allow": [
            "GET"
        ],
        "href": "https://trendmicro2.app.blackduck.com/api/vulnerabilities/BDSA-2021-1709",
        "links": [
            {
                "rel": "cwes",
                "href": "https://trendmicro2.app.blackduck.com/api/cwes/CWE-122"
            },
            {
                "rel": "related-vulnerability",
                "href": "https://trendmicro2.app.blackduck.com/api/vulnerabilities/CVE-2021-30469",
                "label": "NVD"
            },
            {
                "rel": "bdsa-ranges",
                "href": "https://trendmicro2.app.blackduck.com/api/vulnerabilities/BDSA-2021-1709/ranges"
            },
            {
                "rel": "reference",
                "href": "https://sourceforge.net/p/podofo/tickets/129/",
                "label": "ADVISORY"
            },
            {
                "rel": "reference",
                "href": "https://sourceforge.net/p/podofo/tickets/129/",
                "label": "POC"
            },
            {
                "rel": "default-remediation-status",
                "href": "https://trendmicro2.app.blackduck.com/api/vulnerabilities/BDSA-2021-1709/default-remediation-status"
            }
        ]
    }
}